sdk/test/AzureTest/DVRSessionTransport.swift

```diff
@@ -64,6 +64,20 @@ public class DVRSessionTransport: TransportStage {
             .appendingPathComponent("sdk/communication/AzureCommunicationChat/Tests/Recordings").absoluteString else {
             fatalError("SDK Path Invalid")
         }
+        let replacements: [String: Filter.FilterBehavior] = [
+            "authorization": .remove,
+            "client-request-id" : .remove,
+            "retry-after" : .remove,
+            "x-ms-client-request-id": .remove,
+            "x-ms-correlation-request-id": .remove,
+            "x-ms-ratelimit-remaining-subscription-reads": .remove,
+            "x-ms-request-id":.remove,
+            "x-ms-routing-request-id": .remove,
+            "x-ms-gateway-service-instanceid": .remove,
+            "x-ms-ratelimit-remaining-tenant-reads": .remove,
+            "x-ms-served-by": .remove,
+            "x-ms-authorization-auxiliary": .remove
+            ]
```

Comment:
Two things I see here... 
First, replacements isn't being hooked up to anything. 
Second, your SubscriptionIDFilter also isn't hooked up!

This should look like:
```swift
let defaultFilter = Filter()
defaultFilter.filterHeaders = ["authorization": .remove, ...]
session.filters = [SubscriptionIDFilter(), defaultFilter]
```

Reply:
Yea that's what I was talking about when I asked about where we're going with the filters and how much are we leaving open to developers. I was trying to figure out whether or not our goal was to connect it for devs or give them everything so they can connect it themselves.

Reply:
We should provide the default here in DVRSessionTransport. For this PR we don't need to worry about exposing an extension mechanism, but we should wire up the defaults. Extensibility can be a separate PR